src/OpenConext/UserLifecycle/Infrastructure/UserLifecycleBundle/Client/DeprovisionClient.php

```diff
@@ -115,7 +117,15 @@ private function read($path, array $parameters = [])
 
         return $promise->then(
             function (Response $response) use (&$promise, $resource) {
```

Comment:
Did not spot this the other time. But `$promise` is not used in the closure. No need to expose it to the closure in my opinion.